gnd/src/main/res/values/strings.xml

```diff
@@ -159,4 +159,6 @@
   <string name="offline_area_download_started">Download started.</string>
   <string name="offline_area_download_failed">Couldn\'t download area.</string>
   <string name="added_by">Added by %s</string>
+  <string name="reposition_location">Do you want to reposition to this location?</string>
```

Comment:
So far we've been using "point" elsewhere, and you technically can't reposition a location, so let's stick to "point".

Also, I think "Move point" might be more easily understood and less technical-sounding than "Reposition". Sorry I didn't mention this earlier.